### PR TITLE
Allow to install plugins in cli docker image

### PR DESCRIPTION
**Pull request by jakolehm**

### CODE REVIEW

**Comment by jakolehm** [comment]
@kontena/core-platform ping

File: cli/Dockerfile [hunk 1]
```diff
@@ -9,9 +9,12 @@ RUN apk update && \
   ca-certificates libssl1.0 openssl libstdc++ && \
   gem install kontena-cli --no-rdoc --no-ri -v ${CLI_VERSION} && \
   adduser kontena -D -h /home/kontena -s /bin/sh && \
+  echo "gem: --user-install" > /home/kontena/.gemrc && \
   chown -R kontena.kontena /home/kontena && \
   chmod +sx /usr/local/bin/docker
 
+ENV PATH=$PATH:/home/kontena/.gem/ruby/2.3.0/bin
```
Comment (nevalla) [inline cli/Dockerfile]: any chance that this path will change?

Comment (jakolehm) [inline cli/Dockerfile]: it's defined at line 11 .. if someone changes user home directory within this image it does not work anymore.. but still better than broken plugin install?

Comment (nevalla) [inline cli/Dockerfile]: I was thinking about the ruby version but now when thinking harder it won't change.

**Comment by nevalla** [comment]
LGTM!

**Pull request merged**